app/src/main/java/com/example/playlistmaker/library/ui/viewmodel/tracks/LibraryTracksViewModel.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.playlistmaker.library.ui.viewmodel.tracks
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.playlistmaker.library.domain.models.LibraryTrackState
+import com.example.playlistmaker.library.domain.db.LibraryInteractor
+import com.example.playlistmaker.models.Track
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+
+class LibraryTracksViewModel(
+    private val libraryInteractor: LibraryInteractor
+): ViewModel() {
+
+    private val _libraryTracks = MutableStateFlow<LibraryTrackState>(LibraryTrackState.NoInitialized)
+    val libraryTracks: StateFlow<LibraryTrackState> = _libraryTracks
```

Comment:
```suggestion
    val libraryTracks: StateFlow<LibraryTrackState> = _libraryTracks.asStateFlow()
```
Правильно будет вот так использовать, когда хочешь отдать на вью немьютабельный флоу